src/FlipMove.js

```diff
@@ -196,7 +196,9 @@ class FlipMove extends Component {
 
         // We need to take the items out of the "flow" of the document, so that
         // its siblings can move to take its place.
-        removeNodeFromDOMFlow(childData);
+        if (childData.boundingBox) {
+          removeNodeFromDOMFlow(childData);
+        }
```

Comment:
The obvious solution here is to just recalculate the bounding box if it's missing. Sadly, this doesn't work, for reasons I don't understand or have time to dig into; it does recalculate the bounding boxes, but it causes the elements to reshape in weird ways.